GitHubCard/index.js

```diff
@@ -46,10 +70,93 @@ const followersArray = [];
+        cardDiv = document.createElement('div'),
+        userImg = document.createElement('img'),
+        cardInfoDiv = document.createElement('div'),
+        Name = document.createElement('h3'),
+        userName = document.createElement('p'),
+        userLocation = document.createElement('p'),
+        profileParagraph = document.createElement('p'),
+        profileParagraphLink = document.createElement('a'),
+        followersParagraph = document.createElement('p'),
+        followingParagraph = document.createElement('p'),
+        bioParagraph = document.createElement('p'),
+        graphButton = document.createElement('button'),
+        graphDiv = document.createElement('div');
+
+    //Add ClassList
+    cardDiv.classList.add('card');
+    cardInfoDiv.classList.add('card-info');
+    Name.classList.add('name');
+    userName.classList.add('username');
+    graphButton.classList.add('button');
+    graphDiv.classList.add('calendar')
+
+    // Textcontent
+    userImg.src = githubInfo.avatar_url;
+    Name.textContent = githubInfo.name;
+    userName.textContent = githubInfo.login;
+    userLocation.textContent = `Location: ${githubInfo.location}`;
+    profileParagraphLink.textContent = `Profile: ${githubInfo.html_url}`;
+    followersParagraph.textContent = `Followers: ${githubInfo.followers}`;
+    followingParagraph.textContent = `Following: ${githubInfo.following}`
+    bioParagraph.textContent = `Bio: ${githubInfo.bio}`;
+    graphButton.textContent = "SEE GRAPH";
+    graphDiv.textContent = "I am a div for the grahp";
+
+    // graphDiv.style.display = "none";
+    //Append child
+    profileParagraph.appendChild(profileParagraphLink);
+    cardInfoDiv.appendChild(Name);
+    cardInfoDiv.appendChild(userName);
+    cardInfoDiv.appendChild(userLocation);
+    cardInfoDiv.appendChild(profileParagraph);
+    cardInfoDiv.appendChild(followersParagraph);
+    cardInfoDiv.appendChild(followingParagraph);
+    cardInfoDiv.appendChild(bioParagraph);
+    cardInfoDiv.appendChild(graphButton);
+    cardInfoDiv.appendChild(graphDiv);
+    cardDiv.appendChild(userImg);
+    cardDiv.appendChild(cardInfoDiv);
+
+    graphButton.addEventListener('click', (e) => {
+        graphDiv.classList.toggle('toggling');
+    })
+    setTimeout(() => {
+        new GitHubCalendar(`${".calendar"}, ${"adelaadeoye"}`);
+        // GitHubCalendar(`${".calendar"}, ${githubInfo.login}`);
+    }, 3000)
+    return cardDiv;
+}
+
+
+//Followers create function
+
+function followersDataRequest(loginID) {
+    const urlHandler = "https://api.github.com/users/"
+    loginID.forEach(item => {
+        const newUrl = urlHandler + item;
+        axios
+            .get(newUrl)
+            .then(response => {
+                const otherData = response.data;
+                cardContainer.appendChild(createCard(otherData));
+            })
+            .catch(error => {
+                console.log("The data was not returned", error);
+            });
+
+    })
+}
```

Comment:
At the top of this file, you have two axios calls: 1 for your personal user, and one to get your list of followers.

The list of followers then gets passed into this `followersDataRequest` function. Could you refactor this code to remove your first axios call(saving about 10 lines of code), and instead just pass your login into this function?